src/aiortc/mediastreams.py

```diff
@@ -144,3 +146,39 @@ async def recv(self) -> Frame:
         frame.pts = pts
         frame.time_base = time_base
         return frame
+
+
+class VideoNativeStreamTrack(MediaStreamTrack):
```

Comment:
If this is purely for the test suite, move it there. Also the name is not descriptive, what does "native" mean?

Reply:
Please, tell me where I can move this code and tell me a suggestion for rename. I used `native`, because this dummy track send only dummy packets for tests.

Reply:
Since it's only being used by `tests/test_mediastreams.py` (unless I'm mistaken) that sounds like a good place?

Reply:
Ok, I'll use your suggestion.